/**:
  ros__parameters:
    K1 : [10.5, 10.5, 10.5, 0.0, 3.0, 3.0]
    K2 : [20.5, 15.5, 20.5, 0.0, 4.5, 4.5]
    r_b_bg : [0.01, 0.0, 0.02]
    adapt_gain : [0.01, 0.005, 0.01, 0.005, 0.01, 0.005, 0.01, 0.005, 0.01, 0.005, 0.01, 0.005]
    d_gain : [0.03, 0.04, 0.08, 0.03, 0.18, 0.05]
    inertia_matrix : [0.68, 3.32, 3.34]
    mass_matrix : [ 30.0, 0.0, 0.0, 0.0, 0.0, 0.6, 0.0, 30.0, 0.0, 0.0, -0.6, 0.3, 0.0, 0.0, 30.0, 0.6, 0.3, 0.0, 0.0, 0.0, 0.6, 0.68, 0.0, 0.0, 0.0, -0.6, 0.3, 0.0, 3.32, 0.0, 0.6, 0.3, 0.0, 0.0, 0.0, 3.34]
    m : 30.0
